packages/suite/src/components/suite/WordInputAdvanced/index.tsx

```diff
@@ -43,10 +43,10 @@ const WordInputAdvanced = (props: Props) => {
     useEffect(() => {
         const keyboardHandler = (event: KeyboardEvent) => {
             event.preventDefault();
-            if (event.keyCode === 8) {
+            if (event.code === '8') {
```

Comment:
This unfortunately is not that simple 
//enter 
`event.keyCode === '8'`
`event.code === 'Enter'`

Reply:
all other changes need to be updated accordingly as well

Reply:
I think I'm going to use [keycode library](https://www.npmjs.com/package/keycode-js) because they have the keycode constant variables.

Reply:
Sounds good, this way we can get rid of magic numbers/strings

Reply:
I would prefer not to use a library for this. It seems to me that all we need to do is find a proper counterpart for keyCode.

Reply:
@mroz22 I have made updates. Please have a look at it.